src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,31 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (object.getClass().equals(Rectangle.class) && (object != null)) {
```

Comment:
just noticed
will this line work properly if object == null?

Reply:
Yes, it will work properly, I have reakised that after the lection, so I have changed it